userspace/libsinsp/parsers.cpp

```diff
@@ -4908,6 +4931,10 @@ void sinsp_parser::parse_select_poll_epollwait_enter(sinsp_evt *evt)
 	if(evt->m_tinfo->m_lastevent_data == NULL)
 	{
 		evt->m_tinfo->m_lastevent_data = reserve_event_buffer();
+		if(evt->m_tinfo->m_lastevent_data == NULL)
```

Comment:
I think this should be handled like above (ASSERT, log, and return). https://github.com/falcosecurity/libs/pull/888/files#diff-b1e2f413bae0df2c1d50354a44678c2350f5ce108a3609fe14896be2529873f8R889.

Reply:
let me double check these two cases

Reply:
ok this is the same as a `malloc()` failure so we can keep as a `throw()` for consistency, like other ones in the file. Thanks a lot for checking!